packages/neuron-wallet/src/services/transactions.ts

```diff
@@ -217,49 +218,75 @@ export default class TransactionsService {
     return txEntity
   }
 
-  public static generateTx = async (_lockHashes: string[], _targetOutputs: TargetOutput[]) => {
-    const inputs: Input[] = [
-      {
-        previousOutput: {
-          hash: '0xb2becaa4e71e43abc75d1a87280b63df4dceaae1716540faf65e38925d2f641d',
-          index: 0,
-        },
-        args: [],
-        validSince: 0,
-      },
-    ]
-
-    const outputs = [
-      {
-        capacity: '1000',
-        data: '',
+  // system contract info
+  public static contractInfo = async () => {
+    const genesisHash: string = await ckbCore.rpc.getBlockHash(0)
+    const genesisBlock = await ckbCore.rpc.getBlock(genesisHash)
+    const secpTx = genesisBlock.commitTransactions[0]
```

Comment:
secpTx doesn’t describe its meaning well.

Reply:
renamed to `systemScriptTx`